restcomm/restcomm.ui/src/main/webapp/modules/numbers-incoming.html

```diff
@@ -22,14 +22,16 @@
 
         <table ng-show="(numbersList | filter:query).length > 0" class="table table-hover" ng-cloak>
             <thead style="font-size: 18px;">
-            <th class="width-20pc">Number</th>
+             <th class="width-20pc" ng-click="sortBy('phone_number');toggleD=!toggleD;reverse=toggleD;pageChanged()">
+            Number<span class="pull-right" ng-show="predicate === 'phone_number'"><i ng-class="{'fa fa-chevron-up':toggleD,'fa fa-chevron-down':!toggleD}"></i></span></th>
             <th>&nbsp;</th>
-            <th class="width-30pc">Name</th>
-            <th>Features</th>
+            <th class="width-30pc" ng-click="sortBy('friendly_name');toggleC=!toggleC;reverse=toggleC">Name<span class="pull-right" ng-show="predicate === 'friendly_name'"><i ng-class="{'fa fa-chevron-up':toggleC,'fa fa-chevron-down':!toggleC}"></i></span></th>
+      <th>
+    			Features</th>
             <th class="width-10pc">&nbsp;</th>
             </thead>
             <tbody>
-            <tr class="number-row" ng-repeat="pn in filtered = (numbersList | filter:query)">
+            <tr class="number-row" ng-repeat="pn in filtered = (numbersList | filter:query| orderBy:sort:reverse)">
```

Comment:
Replace `sort` by `predicate` which is where the field to sort is stored at (from https://github.com/RestComm/Restcomm-Connect/pull/2211/files#diff-b0cba56bc32531ce8ec4f5174dcd4fe6R97)